src/main/java/dev/meyi/bn/json/Order.java

```diff
@@ -1,12 +1,11 @@
 package dev.meyi.bn.json;
 
-
 import dev.meyi.bn.BazaarNotifier;
 import dev.meyi.bn.json.resp.BazaarItem;
 import dev.meyi.bn.utilities.RenderUtils;
+import net.minecraft.client.Minecraft;
```

Comment:
Can you remove this import?

Reply:
I’ll see if I can, give me a few mins

Reply:
@symt check now, i think it wasnt being used anyway.